lib/WUI/netdev.c

```diff
@@ -105,110 +110,14 @@ void netdev_get_MAC_address(uint32_t netdev_id, uint8_t mac[OTP_MAC_ADDRESS_SIZE
     if (netdev_id == NETDEV_ETH_ID) {
         memcpy(mac, (void *)OTP_MAC_ADDRESS_ADDR, OTP_MAC_ADDRESS_SIZE);
     } else if (netdev_id == NETDEV_ESP_ID) {
-        esp_mac_t tmp;
-        esp_sta_getmac(&tmp, NULL, NULL, 1);
-        memcpy(mac, (void *)&tmp, OTP_MAC_ADDRESS_SIZE);
+        memcpy(mac, wlan0.hwaddr, wlan0.hwaddr_len);
```

Comment:
If I'm correct, this'll solve the issue of the network menu freezing when trying to use an ESP that's not there.